endpoints/openrtb2/video_auction_test.go

```diff
@@ -484,6 +484,50 @@ func TestVideoEndpointValidationsSiteMissingRequiredField(t *testing.T) {
 	assert.Len(t, podErrors, 0, "Pod errors should be empty")
 }
 
+func TestVideoEndpointValidationsMissingVideo(t *testing.T) {
+	ex := &mockExchangeVideo{}
+	deps := mockDeps(t, ex)
+	deps.cfg.VideoStoredRequestRequired = true
+
+	durationRange := []int{
+		15,
+		30,
+	}
+
+	pods := []openrtb_ext.Pod{
+		{
+			PodId:            1,
+			AdPodDurationSec: 30,
+			ConfigId:         "qwerty",
+		},
+		{
+			PodId:            2,
+			AdPodDurationSec: 30,
+			ConfigId:         "qwerty",
+		},
+	}
+
+	req := openrtb_ext.BidRequestVideo{
+		StoredRequestId: "123",
+		PodConfig: openrtb_ext.PodConfig{
+			DurationRangeSec:     durationRange,
```

Comment:
Nitpick: Last minor nitpick here. Since durationRange is only used to construct the config, consider declaring it inline like:
```
PodConfig: openrtb_ext.PodConfig{
			DurationRangeSec:     []int{15,30},
```

Reply:
Updated to be inline. Also moved pods declaration to be inline for consistency.